lib/jsonrpc/server.go

```diff
@@ -53,7 +53,7 @@ func (s *RPCServer) handleWS(ctx context.Context, w http.ResponseWriter, r *http
 		conn:    c,
 		handler: s.methods,
 		exiting: make(chan struct{}),
-	}).handleWsConn(ctx)
+	}).handleWsConn(ctx, ReconnectConfig{})
```

Comment:
Since focused on client use, at the point of seeing this all the change was done.
Not quite like it tbh.